src/libraries/System.Text.Json/gen/Helpers/KnownTypeSymbols.cs

```diff
@@ -341,14 +341,14 @@ public bool IsImmutableDictionaryType(ITypeSymbol type, out string? factoryTypeF
         private INamedTypeSymbol? GetOrResolveType(Type type, ref Option<INamedTypeSymbol?> field)
             => GetOrResolveType(type.FullName!, ref field);
 
-        private INamedTypeSymbol? GetOrResolveType(string fullyQualifiedName, ref Option<INamedTypeSymbol?> field)
+        private INamedTypeSymbol? GetOrResolveType(string GlobalDisplayString, ref Option<INamedTypeSymbol?> field)
```

Comment:
Doesn't FQN better emphasize that it's a type's identifier? Also, naming style:
```suggestion
        private INamedTypeSymbol? GetOrResolveType(string globalDisplayString, ref Option<INamedTypeSymbol?> field)
```